### ISSUE TITLE
init creates a new package.json with an older version

### ISSUE THREAD

**Issue by miketheman** [amzn/style-dictionary#424]
Hello!

I was trying this out today, and found this behavior confusing:

1. Create a new package.json in an empty directory
1. Run `yarn add --dev style-dictionary` - get latest version 2.10.0
1. Run `yarn run style-dictionary init complete` - the package.json gets templated value:

```json
...
  "devDependencies": {
    "style-dictionary": "2.9.0"
  }
...
```

I would expect that the generated template match the version installed - unless this is some other behavior?

**Comment by dbanksdesign** [comment]
This is definitely a bug. Looks like a botched release on my part, the package.json in the complete example did not get updated in the last release: https://github.com/amzn/style-dictionary/blob/master/examples/complete/package.json#L14

**Comment by chazzmoney** [comment]
@dbanksdesign this is still in the codebase, lol...

Can we just set to '*'?

**Comment by dbanksdesign** [comment]
We could.... I don't know if that is a bad practice or not, I haven't seen '*' in very many package.json files. Let's take a look at how some other frameworks handle this.

**Comment by chazzmoney** [comment]
I mean, we could add it to the process for creating a tagged release... but it would be better if that was automated

**Comment by dbanksdesign** [comment]
We have a pre-version script that should bump up all the version numbers of the examples, but it might be broken... If we moved the repository to a monorepo with each example as its own package then I think this could be handled nicer without a custom script... Maybe something to look in to.

**Comment by dbanksdesign** [comment]
Fixed this in #755
